clarify.toml

```diff
@@ -1,3 +1,10 @@
+[spdx]
+ignore-licenses = [
+    # Apache-2.0 is often misclassified as Pixar, which is a significantly more rare
+    # https://github.com/jpeddicord/askalono/issues/94
+    "Pixar"
+]
```

Comment:
This is to fix the failure when running `make brupop-image`. The `license-scan` was not happy with the Pixar license.

```
1.384 Error: detected license "Pixar" from LICENSE-APACHE is not present in the license field "MIT OR Apache-2.0" for allocator-api2
```
Full log for the CI run - https://github.com/bottlerocket-os/bottlerocket-update-operator/actions/runs/12838717210/job/35804816033?pr=709.

Referring to [this code](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/331/files#diff-ae237b2dd44b0855242f2349cfe51b15b570d4ba0470b0b84bd24e5f3820d07a) in core-kit for the fix.